.github/workflows/ci.yml

```
@@ -15,6 +15,9 @@
       - name: Setup
         uses: ./.github/actions/setup
 
+      - name: Generate types
+        run: pnpm typegen
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)